client/src/components/Hamburger/Hamburger.stories.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react'; 
+import Hamburger from './Hamburger';
+
+let burger = "hamburgerBars";
+let icons = "bars";
+
+function switchMenu() {
+    // const [burger, setBurger] = useState("hamburgerTimes");
+    const [icons, setIcons] = useState("times");
+}
+
+function switchBack() {
+    // const [burger, setBurger] = useState("hamburgerBars");
+    const [icons, setIcons] = useState("bars");
+}
+
+function handleClick() {
+    console.log(burger)
+    if(burger == "hambugerBars" && icons == "bars"){
+    return(switchMenu)
+    }
+    else{
+    return(switchBack)
+    
+    }
+    
+}
```

Comment:
Can you try to move some of this logic into the component?

You have the right idea here and this does look waaaaaaay better! The `handleClick` function should toggle some variable true/false. Then your `Hamburger` component should have a prop that takes true or false to determine whether or not it's closed (so three bars) or open (so an X).

Reply:
The logic should then live in your component